### DISCUSSION TITLE
manually input current segment's start time (or end time) - timecode format change to use frames?

### DISCUSSION THREAD

**Discussion by fyhtma** [mifi/lossless-cut#1960] [Support] [answered]
Excuse me, how can I change the timecode format in feature" manually input current segment's start time" (and "manually input current segment's end time") ?

I want to use HH:MM:SS;FF timecode format
![wanted](https://github.com/mifi/lossless-cut/assets/52983487/f42faf5d-677c-4ab6-8046-918ccc49c6c5)

not HH:MM:SS.sss as currently
![losslesscut](https://github.com/mifi/lossless-cut/assets/52983487/d323e599-3579-4272-858e-70283853984b)

**Comment by mifi** [comment, marked as answer]
currently not possible, but i will add it in the next version
